hardware/lcd/config.in

```diff
@@ -31,6 +31,13 @@ mainmenu_option next_comment
       define_bool I2C_MASTER_SUPPORT y
       define_bool I2C_PCF8574X_SUPPORT y
     fi
+
+    if [ "$HD44780_I2CSUPPORT" = "y" ]; then
+      if [ "$HD44780_TYPE" = "HD44780_WDC2704" ]; then
+        define_string CONF_HD44780_TYPE "HD44780_WDC2704"
```

Comment:
This is only one of many possible CONF_HD44780_TYPE.. Please take care of the others by providing this one for the supported combination. And please do not mix the differnt HD44780-style display-types with their connection.